types/types.go

```diff
@@ -218,32 +225,33 @@ type GooglechatConfig struct {
 
 // Statistics is a struct to store stastics
 type Statistics struct {
-	Requests      *expvar.Map
-	FIFO          *expvar.Map
-	GRPC          *expvar.Map
-	Falco         *expvar.Map
-	Slack         *expvar.Map
-	Mattermost    *expvar.Map
-	Rocketchat    *expvar.Map
-	Teams         *expvar.Map
-	Datadog       *expvar.Map
-	Discord       *expvar.Map
-	Alertmanager  *expvar.Map
-	Elasticsearch *expvar.Map
-	Loki          *expvar.Map
-	Nats          *expvar.Map
-	Influxdb      *expvar.Map
-	AWSLambda     *expvar.Map
-	AWSSQS        *expvar.Map
-	AWSSNS        *expvar.Map
-	SMTP          *expvar.Map
-	Opsgenie      *expvar.Map
-	Statsd        *expvar.Map
-	Dogstatsd     *expvar.Map
-	Webhook       *expvar.Map
-	AzureEventHub *expvar.Map
-	GCPPubSub     *expvar.Map
-	GoogleChat    *expvar.Map
+	Requests          *expvar.Map
+	FIFO              *expvar.Map
+	GRPC              *expvar.Map
+	Falco             *expvar.Map
+	Slack             *expvar.Map
+	Mattermost        *expvar.Map
+	Rocketchat        *expvar.Map
+	Teams             *expvar.Map
+	Datadog           *expvar.Map
+	Discord           *expvar.Map
+	Alertmanager      *expvar.Map
+	Elasticsearch     *expvar.Map
+	Loki              *expvar.Map
+	Nats              *expvar.Map
+	Influxdb          *expvar.Map
+	AWSLambda         *expvar.Map
+	AWSSQS            *expvar.Map
+	AWSSNS            *expvar.Map
+	AWSCLOUDWATCHLOGS *expvar.Map
```

Comment:
I have one question 🙇 
How about using camel case like other fields(e.g. `AWSLambda`)?

```suggestion
	AWSCouldWatchLogs *expvar.Map
```